### ISSUE TITLE
Question with Sponge EntityTypes + Woot

### ISSUE THREAD

**Issue by YOYOK9** [Ipsis/Woot#400]
So this is just a question about the woot + sponge issues that have been happening. Was playing around with the dragon and noticed this when I ran "/sponge entityinfo"  on the dragon*
(first is enderman, 2nd+3rd are dragon)
https://puu.sh/C8DJu/2f56003688.jpg

Now not sure of a command to test that without sponge, however shouldn't the entity type be "minecraft:ender_dragon"? just curious if this would cause issues? 

Note*
Only the dragon comes up with the minecraft:complexpart as it's entity type. 
Tried it on a few other mobs as well:
https://puu.sh/C8F4K/28d3e0429b.jpg

**Comment by Ipsis** [comment]
minecraft:ender_dragon is the name that comes up with vanilla and works correctly.
If you are seeing any issues then I'm afraid it is down to the whole Sponge problem.
The last time I checked the Ender Dragon capture was still outstanding with Sponge.

Can I assume that the issue that you are seeing is that you cannot program a shard for the EnderDragon?

**Comment by YOYOK9** [comment]
yeah... sadly been following the shard stuff for a while now. was just poking around to see if I could find a way to "trick" it into working. Should I pass this weird complexpart off to sponge do you think?

**Comment by Ipsis** [comment]
If you really want a shard - "/woot give <playername> minecraft:ender_dragon EnderDragon"
As long as you have are running latest version of Woot and have OP.

The Sponge bug is "SpongePowered/SpongeForge#1912"

I might see if there is a way to add a recipe for the EnderDragon to bypass this issue. The only thing is that I don't think that the looting works properly with Sponge either.

**Comment by YOYOK9** [comment]
Yeah I have tried that way in single player. Sadly the servers I have seen do not want to use that and are just waiting for sponge to fix it. Which we all know could take a long time...
